composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^7.1",
         "ext-pdo": "*",
         "doctrine/collections": "^1.4",
-        "doctrine/dbal": ">=2.5-dev,<2.7-dev",
+        "doctrine/dbal": "^2.6",
```

Comment:
Did we allow >=2.7-dev intentionally here or is it just accidental?

Reply:
I set it up intentionally for look at tests and they were passed, so I think it's ok.

Reply:
I think it was accidental =) It should indeed be `>=2.6-dev,<2.7-dev` or even `>=2.6-dev,<2.8-dev`

Reply:
With how carefully we're checking BC, does it even matter? Might as well use the semver operator here.

Reply:
That's true... @Majkl578 @Ocramius anything against that?